.github/workflows/test.yml

```
@@ -0,0 +1,217 @@
+        run: |
+          # shellcheck disable=SC1090
+          . ~/.bashrc
+          # shellcheck disable=SC1090
+          . ~/.bash_profile
+
+          mamba activate test
+          pip install -U ./partcad ./partcad-cli
+          mamba deactivate
+
+  test-examples-partcad:
+    runs-on: ${{ matrix.os }}
```

actionlint (1.7.4)

property "os" is not defined in object type {}

(expression)